src/rust/ide/view/graph-editor/src/component/visualization/path.rs

```diff
@@ -4,6 +4,7 @@ use crate::prelude::*;
 
 use crate::data;
 
+use fmt::Formatter;
```

Comment:
I'd ask you to remove it and use `fmt::Formatter` instead in the use place. Less imports and there is just a single usage of that